mm2src/mm2_main/tests/docker_tests_main.rs

```diff
@@ -100,6 +102,27 @@ pub fn docker_tests_runner(tests: &[&TestDescAndFn]) {
     test_main(&args, owned_tests, None);
 }
 
+fn wait_for_geth_node_ready() {
+    let mut attempts = 0;
+    loop {
+        if attempts >= 5 {
+            println!("Failed to connect to Geth node after several attempts.");
+            break;
+        }
+        match block_on(GETH_WEB3.eth().block_number()) {
+            Ok(block_number) => {
+                println!("Geth node is ready, latest block number: {:?}", block_number);
+                break;
+            },
+            Err(e) => {
+                println!("Failed to connect to Geth node: {:?}, retrying...", e);
+                attempts += 1;
+                thread::sleep(Duration::from_secs(5));
```

Comment:
Yeah you are right, total 25 seconds are too much. Changed to 10 attempts and 1 second duration. Ive set duration to 1 second, because usually the first few seconds are enough. and this will also prevent unnecessary log entries when the process is successful.